package.json

```diff
@@ -143,6 +143,11 @@
               "verbose"
             ],
             "description": "Traces the communication between VS Code and the SourceKit-LSP language server."
+          },
+          "swiftpm.testDiscovery.always": {
+            "type": "boolean",
```

Comment:
Do we really need the config option?

Reply:
There’s still someone who prefers using `LinuxMain` (e.g. for CI compatibility), and I think we’d better offer such choice.

Reply:
Fair point. Wondering how much work we put into supporting earlier versions of swift.

Reply:
I understand what it is for. I just don't think it is something we need to support. I don't want to clutter the configuration with settings that get used by virtually no one.